client/my-sites/stats/feedback/use-site-types.ts

```diff
@@ -0,0 +1,19 @@
+import { useSelector } from 'calypso/state';
+import isVipSite from 'calypso/state/selectors/is-vip-site';
+import { isJetpackSite } from 'calypso/state/sites/selectors';
+import getSiteOption from 'calypso/state/sites/selectors/get-site-option';
+
+function useSiteTypes( siteId: number ) {
+	const isJetpackNotAtomic = useSelector(
+		( state ) => !! isJetpackSite( state, siteId, { treatAtomicAsJetpackSite: false } )
+	);
+	const isVip = useSelector(
+		( state ) =>
+			!! isVipSite( state as object, siteId as number ) ||
+			!! getSiteOption( state, siteId, 'is_vip' )
+	);
+
+	return { isJetpackNotAtomic, isVip };
```

Comment:
Should we expand this to pull in all the site type info? isAtomic, isWpcom, isP2, etc?

We could pull the various checks from `/stats/stats-notices/index.tsx` into this hook and share it from `/stats/hooks/` as the various checks are likely to prove useful in other areas too.